### PR TITLE
bn-models:

### PR DESCRIPTION
**Pull request by sethgerou-noaa**
fishery, person-alias, and ots-target documentation

### CODE REVIEW

**Review by melinashak-noaa — COMMENTED** [review]
overall looks good, just some few comments

File: bn-models/src/models/_lookups/fishery.ts [hunk 1]
```diff
@@ -2,82 +2,92 @@ import { BaseLookup } from '../_base';
...
+      lookupId: 586,
+      lookupType: "FISHERY"
+    }
+  }
+
+*/
+
+
+
+/* Notes from Will - where do these belong?
```
[the remark this answers is not in the document]
Comment (sethgerou-noaa) [reply bn-models/src/models/_lookups/fishery.ts:56]: yes, will do.

File: bn-models/src/models/_lookups/fishery.ts [hunk 2]
```diff
@@ -2,82 +2,92 @@ import { BaseLookup } from '../_base';
 
 // TODO Full implementation
 export const FisheryTypeName = 'fishery';
-export type Sector = string; // WCGOP Obs Analyst Sectors
-export type GovernmentOrganization = string; // TODO LOOKUPS
-// NOAA Fisheries, ODFW, WDFW, CDFW, IPHC
+export type Sector = object; // sector lookups
+export type GovernmentOrganization = string;
```
[the remark this answers is not in the document]
Comment (sethgerou-noaa) [reply bn-models/src/models/_lookups/fishery.ts:6]: yeah, this should probably be an enum, as there are only a few valid options.  On the other hand, it's part of a lookup, so not like someone is making a fishery.governmentOrganization selection from a form.  Will make it an enum for now.

File: bn-models/src/models/_lookups/fishery.ts [hunk 3]
```diff
@@ -2,82 +2,92 @@ import { BaseLookup } from '../_base';
 
 // TODO Full implementation
 export const FisheryTypeName = 'fishery';
-export type Sector = string; // WCGOP Obs Analyst Sectors
-export type GovernmentOrganization = string; // TODO LOOKUPS
-// NOAA Fisheries, ODFW, WDFW, CDFW, IPHC
+export type Sector = object; // sector lookups
```
[the remark this answers is not in the document]
Comment (sethgerou-noaa) [reply bn-models/src/models/_lookups/fishery.ts:5]: good point.  there is an existing fishery-sector model in the trips-api directory - until recently, I didn't realize fishery-sector was used outside em.  Moving it to lookups and integrating into the fishery model.

File: bn-models/src/models/_lookups/person-alias.ts [hunk 4]
```diff
@@ -4,12 +4,31 @@ import { CouchID } from '../_common';
 export const PersonAliasTypeName = 'person-alias';
```
[the remark this answers is not in the document]
Comment (sethgerou-noaa) [reply bn-models/src/models/_lookups/person-alias.ts:5]: yep, adding.  Also adding a description to ots-target model.

File: bn-models/src/models/_lookups/fishery.ts (hunk 3 above)
[the remark this answers is not in the document]
Comment (melinashak-noaa) [reply bn-models/src/models/_lookups/fishery.ts:50]: Looks good, can delete this line of code now. Checked and don't see any files importing sector so believe is should be safe to delete.

File: bn-models/src/models/_lookups/fishery.ts (hunk 2 above)
[the remark this answers is not in the document]
Comment (melinashak-noaa) [reply bn-models/src/models/_lookups/fishery.ts:51]: can delete this line of code as well. see governmentOrganization imported by permit.ts so might need to make a few changes there to make sure things still work there

**Review by melinashak-noaa — APPROVED** [review]
Changes look good, just delete a few lines of code and should be good to commit.

**Comment by sethgerou-noaa** [comment]
@melinashak-noaa  good catches, thanks!

**Pull request merged**